### DISCUSSION TITLE
How to change the background color of a individual page independent of the dark mode or dark plugin variables?

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#13789] [General - Components / Directives / etc] [answered]
How to change the background color of a individual page independent of the dark mode or dark plugin variables? 

I mean: When ever a user change his or her dark mode, I want some individual pages to keep their original background color which I set in the <style scoped> section of the page. Is this achievable, if yes, how?

**Comment by metalsadman** [comment, marked as answer]
you can check Dark plugin's `isActive` prop. https://quasar.dev/quasar-plugins/dark#watching-for-status-change
you can also check it in the body https://quasar.dev/style/dark-mode#introduction
